forge-gui/res/cardsfolder/upcoming/long_goodbye.txt

```diff
@@ -0,0 +1,6 @@
+Name:Long Goodbye
+ManaCost:1 B
+Types:Instant
+K:This spell can't be countered.
```

Comment:
this KW no longer exists

Reply:
Fixed. Used a reference dated prior to this update. Thanks!